File: 0026-Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) < 2:
            return len(nums)
        rst = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i-1]:
                nums[rst] = nums[i]
                rst += 1
        return rst

File: 0026-Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
       if(nums.size()<2){
           return nums.size();
       }
        int rst = 1;
        for(int i=1;i<nums.size();i++){
            if(nums[i] != nums[i-1]){
                nums[rst++] = nums[i];
            }
        }
        return rst;
        
    }
};